ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/worldview/VerkleWorldState.java

```diff
@@ -233,6 +233,23 @@ private void generateAccountValues(
                 hashAndSavePreImage(accountKey), updatedAcount.serializeAccount()));
   }
 
+  private void handleCoupledCodeAccountUpdates(
+      final Address accountKey,
+      final VerkleEntryFactory verkleEntryFactory,
+      final DiffBasedValue<VerkleAccount> accountUpdate) {
+    final VerkleAccount priorAccount = accountUpdate.getPrior();
+    final VerkleAccount updatedAccount = accountUpdate.getUpdated();
+    if (priorAccount == null) {
+      verkleEntryFactory.generateCodeHashKeyValueForUpdate(
+          accountKey, updatedAccount.getCodeHash());
+      return;
+    }
+    Bytes currentCode = get(accountKey).getCode();
```

Comment:
why we are not using accumulator to retrieve the code instead of reading from database ?